src/app/components/Application/Application.jsx

```diff
@@ -86,18 +82,27 @@ class Application extends React.Component {
     Store.unlisten(this.onChange);
   }
 
-  onChange() {
-    this.setState({ data: Store.getState() });
-  }
+  onWindowResize() {
+    const { media } = this.state;
+    const style = {
+      xtrasmallBreakPoint: 483,
```

Comment:
A /veeerrry general/ code style nit:  Generally you want these to be defined as consts either at the beginning of the document, or (better yet), a separate `constants` file.  The idea is that if these need to be changed it is super easy to find them and change them.  (The other idea is that if you ever need this anywhere else, it's easy to import every time you need breakpoints they clearly only live in one place.)